python/paddle/v2/fluid/tests/book/test_label_semantic_roles.py

```diff
@@ -211,12 +219,75 @@ def main():
                 if batch_id != 0:
                     print("second per batch: " + str((time.time() - start_time)
                                                      / batch_id))
-
-            # exit early for CI
-            exit(0)
+                # Set the threshold low to speed up the CI test
+                if float(pass_precision) > 0.05:
+                    if save_dirname is not None:
+                        fluid.io.save_inference_model(save_dirname, [
+                            'word_data', 'verb_data', 'ctx_n2_data',
+                            'ctx_n1_data', 'ctx_0_data', 'ctx_p1_data',
+                            'ctx_p2_data', 'mark_data'
+                        ], [feature_out], exe)
+                    return
 
             batch_id = batch_id + 1
 
 
+def infer(save_dirname=None):
+    if save_dirname is None:
+        return
+
+    # place = fluid.CPUPlace()
+    place = fluid.CUDAPlace(0)
```

Comment:
maybe something like this: `place = fluid.CUDAPlace(0) if use_cuda else fluid.CPUPlace()`

Reply:
Good point! Done.